README.md

```diff
@@ -44,57 +80,33 @@ A wide range of lockfiles are supported by utilizing this [lockfile package](htt
 - `pom.xml`\*         
 - `requirements.txt`\*
 
-### Example
+#### Example
 
 ```bash
 $ go run ./cmd/osv-scanner --lockfile=/path/to/your/package-lock.json -L /path/to/another/Cargo.lock
 ```
 
-## Scanning a Debian based docker image packages
+### Scanning a Debian based docker image packages
 
 This tool will scrape the list of installed packages in a Debian image and query for vulnerabilities on them.
 
 Currently only Debian based docker image scanning is supported.
 
 Requires `docker` to be installed and the tool to have permission calling it.
 
-### Example
+#### Example
 
 ```bash
 $ go run ./cmd/osv-scanner --docker image_name:latest
 ```
 
-## Scanning a directory
-
-This tool will walk through a list of directories to find:
-- Lockfiles
-- SBOMs
-- git directories for the latest commit hash
-
-and make requests to OSV to determine affected vulnerabilities.
-
-You can have it recursively walk through subdirectories with the `--recursive` / `-r` flag.
-
-Searching for git commit hash is intended to work with projects that use
-git submodules or a similar mechanism where dependencies are checked out
-as real git repositories.
-
-### Example
-
-```bash
-$ go run ./cmd/osv-scanner -r /path/to/your/dir
-```
-
-## Configure `osv-scanner`
+## Configure OSV-Scanner
 
-By placing a `osv-scanner.toml` file in any parent directory of the file being
-scanned will be used to configure scanning of that file. This can be overridden
-by passing the `--config=/path/to/config` flag.
+To configure scanning, place an osv-scanner.toml file in the scanned file's directory. To override this osv-scanner.toml file, pass the `--config=/path/to/config.toml` flag with the path to the configuration you want to apply instead.
 
 Currently, there is only 1 option to configure:
 ### Ignore vulnerabilities by ID
-Vulnerabilities can be marked as ignored by putting the ID an entry
-under the `IgnoreVulns` key, along with optional reason and expiry date.
+To ignore a vulnerability, enter the ID under the `IgnoreVulns` key. Optionally, add an expiry date or reason.
```

Comment:
Also asked this earlier today but wanted to clarify again:

If we have the following directory structure:

```
foo/osv-scanner.toml
foo/package-lock.json
bar/osv-scanner.toml
bar/package-lock.json
osv-scanner.toml
package-lock.json
```

Will scanning:

- `foo/package-lock.json` use `foo/osv-scanner.toml` by default? 
- `bar/package-lock.json` use `bar/osv-scanner.toml` by default? 
- `package-lock.json` use `osv-scanner.toml` by default?

Reply:
Yep, that's correct